src/components/timeline/Track.css

```diff
@@ -38,20 +38,22 @@
   box-sizing: border-box;
   flex-flow: row nowrap;
   align-items: center;
-  padding-left: 14px;
   border-right: 1px solid var(--grey-30);
   cursor: default;
 }
 
 .timelineTrackNameButton {
   overflow: hidden;
-  padding: 0;
-  margin: 0;
+  width: calc(100% - 8px);
+  height: calc(100% - 8px);
```

Comment:
nit: please add a comment explaining that these 8px is to allow for the box-shadow of the focus style.